tests/unit/examples/test_serving_an_xgboost_model_with_merlin_systems.py

```diff
@@ -34,5 +34,5 @@ def test_example_serving_xgboost(tb):
     )
     tb.execute_cell(list(range(0, 14)))
 
-    with run_triton_server("ensemble"):
+    with run_triton_server("ensemble", grpc_port=8001):
```

Comment:
Would hard-coding triton's default port number here re-introduce the issue that the PR is trying to avoid?

Reply:
So the tests have now passed with port 8001 set as default which is what was running with before. I'm tempted to say let's keep the port set to 8001 and see how often we see the new exception (added in this PR) about triton already running show up. We can then change the default to 0 (for random port selection) if we see it happening too much.